### ISSUE TITLE
generator-star-spacing: false positive at computed methods

### ISSUE THREAD

**Issue by mysticatea** [eslint/eslint#6135]
**What version of ESLint are you using?**

v2.9.0

**What parser (default, Babel-ESLint, etc.) are you using?**

default.

**Please show your full configuration:**

nothing.

**What did you do? Please include the actual source code causing the issue.**

```js
/*eslint-env es6*/
/*eslint generator-star-spacing:[2, "after"]*/

class A {
    * foo() {
    }
    
    * [Symbol.iterator]() {
    }
}
```

**What did you expect to happen?**

No errors.

**What actually happened? Please include the actual, raw output from ESLint.**

`generator-star-spacing` rule is warning at `* [Symbol.iterator]() {`

![image](https://cloud.githubusercontent.com/assets/1937871/15176648/0e26ab24-17a7-11e6-891b-205792d4ec65.png)

**Comment by mysticatea** [comment]
I'm working on this.
